BraveWallet/Crypto/Stores/AssetDetailStore.swift

```diff
@@ -94,22 +92,22 @@ class AssetDetailStore: ObservableObject {
       }
       assetRatioService.price(
         [token.symbol],
-        toAssets: ["usd", "btc"],
+        toAssets: [currencyFormatter.currencyCode, "btc"],
```

Comment:
Does this need to be normalized to lowercase for the API?

Reply:
They're lowercased internally in asset ratio service, we could send as lowercase to be safe but shouldn't be needed.

Price: https://github.com/brave/brave-core/blob/master/components/brave_wallet/browser/asset_ratio_service.cc#L163
Price history: https://github.com/brave/brave-core/blob/master/components/brave_wallet/browser/asset_ratio_service.cc#L207